local_settings_example.py

```diff
@@ -7,13 +7,24 @@
 class LocalMixin:
     """Add custom local settings here."""
 
-    SOCIAL_AUTH_TUNNISTAMO_ALLOWED_REDIRECT_HOSTS = ["localhost:3000", "localhost:3001"]
+    # On macOS, you might need to set these paths manually (https://gis.stackexchange.com/a/416882)
+    # GDAL_LIBRARY_PATH = "/opt/homebrew/lib/libgdal.dylib"
+    # GEOS_LIBRARY_PATH = "/opt/homebrew/lib/libgeos_c.dylib"
+
+    # CELERY_ENABLED = False  # If False, run tasks synchronously without a worker, but scheduled tasks will run.
     CELERY_TASK_ALWAYS_EAGER = True
 
+    GRAPHQL_CODEGEN_ENABLED = False  # Set to True to enable codegen for frontend by disabling CSRF protection
```

Comment:
Frontend devs don't really use Local interpreters, so this should preferably be in `DockerMixin`.